package.json

```diff
@@ -44,6 +44,7 @@
     "archiver": "^5.3.1",
     "aws-sdk": "^2.1393.0",
     "axios": "^0.22.0",
+    "caniuse-lite": "^1.0.30001616",
```

Comment:
is this intentional? I welcome it if it can make the builds faster!

Reply:
I don't know if this made the builds faster, but eslint wanted it, and I think it's made linting more responsive for me.